src/components/NavBar/MenuItemMobile/index.tsx

```diff
@@ -0,0 +1,184 @@
+            IconButtonProps={{ color: 'inherit', size: 'small' }}
+            aria-controls={title}
+            id={title}
+          >
+            <img className={classes.icon} src={icon} alt={title} />
+            <Typography variant="body2">{title}</Typography>
+          </AccordionSummary>
+          <AccordionDetails className={classes.accordionDetail}>
+            <Grid container direction="column">
+              {layersCategories.map(
+                ({ title: categoryTitle, layers, tables }) => (
+                  <Grid
+                    item
+                    key={categoryTitle}
+                    className={classes.categoryContainer}
+                  >
+                    <Typography
+                      variant="body2"
+                      className={classes.categoryTitle}
+                    >
+                      {categoryTitle}
+                    </Typography>
+                    <hr />
+
+                    {layers.map(layer => {
+                      const { id: layerId, title: layerTitle } = layer;
+                      const selected = Boolean(
+                        selectedLayers.find(
+                          ({ id: testId }) => testId === layerId,
+                        ),
+                      );
+                      return (
+                        <div key={layerId} className={classes.layersContainer}>
+                          <Switch
+                            size="small"
+                            color="default"
+                            checked={selected}
+                            onChange={() => toggleLayerValue(selected, layer)}
+                            inputProps={{ 'aria-label': layerTitle }}
+                          />{' '}
+                          <Typography variant="body1">{layerTitle}</Typography>
+                        </div>
+                      );
+                    })}
+
+                    {tables.map(table => (
+                      <Button
+                        id={table.title}
+                        key={table.title}
+                        onClick={() => showTableClicked(table)}
+                      >
+                        <Typography variant="body1">{table.title}</Typography>
+                      </Button>
+                    ))}
+                  </Grid>
+                ),
+              )}
+            </Grid>
+          </AccordionDetails>
+        </Accordion>
+      ))}
+    </>
+  );
+}
+
+const styles = (theme: Theme) =>
+  createStyles({
+    icon: {
+      width: 18,
+      marginRight: 6,
+    },
+
+    categoryContainer: {
+      marginBottom: 16,
+      '&:last-child': {
+        marginBottom: 0,
+      },
+    },
+
+    categoryTitle: {
+      fontWeight: 'bold',
+      textAlign: 'left',
+    },
+
+    layersContainer: {
+      display: 'flex',
+      marginBottom: 8,
+    },
+
+    accordion: {
+      boxShadow: 'none',
+      '&.Mui-expanded': {
```

Comment:
Could we use this to the global MUI theme? If we aren't using Accordion anywhere else, this might be cleaner/less prone to breaking.

Reply:
just learn that I can use `elevation={0}` so I removed the `boxShadow: none` for this one

Reply:
we should always prioritize using MUI props over classes. If `elevation` works fine definitely use that over a CSS style.

Reply:
Reminder: we shouldn't have to do `&.Mui-expanded`. Try doing it in the MUI global theme if you really need this style.
To do this, go here:
https://github.com/WFP-VAM/prism-frontend/blob/ca955c5cc5d3088d591b0d05719be0f604116231/src/muiTheme.tsx#L59
and add this: 
![image](https://user-images.githubusercontent.com/25571542/111622130-eb2de180-883c-11eb-8a9c-6d988a45965e.png)